process_annotation_to_dict.py

```diff
@@ -105,16 +106,24 @@ def describe_continuous(df: pd.DataFrame) -> dict:
     }
 
 
+def get_missing(df: pd.DataFrame) -> bool:
+    return [row["value"] for rid, row in df.iterrows() if row["controlled_term"] == "nb:MissingValue"]
+
+
 def describe_discrete(df: pd.DataFrame) -> dict:
-    return {
+    col_annotation = {
         "Annotations": {
             **describe_isabout(get_col_rows(df)["controlled_term"].item()),
             "Levels": {
                 row.value: describe_level(row.controlled_term)
-                for _, row in get_level_rows(df).iterrows()
+                for _, row in get_level_rows(df).iterrows() if not row["controlled_term"] == "nb:MissingValue"
```

Comment:
Question: do we need to add this explicit check for missing values for the tool / continuous column types as well?

According to our current data dictionary [docs](https://www.neurobagel.org/documentation/dictionaries/#missing-values) + [reference example .json](https://github.com/neurobagel/bagel-cli/blob/main/bagel/tests/data/example_synthetic.json), any phenotypic column that's not an identifier column can have annotated missing values (I believe the CLI allows this as well). Based on the `bulk_annotation_levels` gsheet, I see that there's at least one assessment tool (e.g., ds003823 "ses-pre_task-SART" column) with a missing value annotated. Not sure how/if these are currently handled.

Reply:
Yes, you are correct. This is a big oversight. in fact the assessment tool workflow has to have missing_values defined because that's the way we figure out who "does not" have the tool. I believe that at the moment we erroneously interpret that every subject has the assessment because we do not model this.

Reply:
After some thinking and discussion: let's make an issue to address the fact that we aren't handling missing values for annotations of tools and of continuous values (i.e. age) at the moment. For age at least, this will involve a change to how we do annotations, and so we should have a discussion before we make this change